src/scripts/test_python.py

```diff
@@ -718,6 +719,66 @@ def test_srp6(self):
         key_s = server.step2(a)
         self.assertEqual(key_c, key_s)
 
+
+class BotanPythonZfecTests(unittest.TestCase):
+    """
+    Tests relating to the ZFEC bindings
+    """
+
+    def test_encode(self):
+        """
+        Simple encoder test.
+
+        Could benefit from more variations
+        """
+        n = 3
+        k = 2
+        input_bytes = b"abcdefgh" + b"ijklmnop"
+        output_shares = botan2.zfec_encode(k, n, input_bytes)
```

Comment:
These tests are fine, but they need to be adapted to the new import. Specifically here in the tests we are importing `botan3` as `botan` (so we don't have to modify all of the tests every time we bump the major version). This is currently causing some CI failures.

Reply:
Fixed in 7136dd7ce